mpifileutils.spec

```
@@ -250,9 +250,9 @@ done
 %endif
 
 %changelog
-* Tue Feb 22 2022 Dalton A. Bohning <daltonx.bohning@intel.com> - 0.11.1-1
-- Update to v0.11.1
-- Remove patch2 in favor of upstream d5f41b3
+* Wed Feb 23 2022 Dalton A. Bohning <daltonx.bohning@intel.com> - 0.11.1-1
+- Update to v0.11.1 + patch 0837021e841e04475e804d8989d000887d5bb576
+- Remove patch2 in favor of upstream d5f41b3, containe in v0.11.1
```

```suggestion
- Remove patch2 which was merged into v0.11.1
```
mostly just to fix the speeling error.  But I think is more clear about why patch2 is removed.

---

Thanks, fixed. Though some issues were found in CI, so I'll have to wait another build + CI test cycle